paper_trail.gemspec

```diff
@@ -56,6 +56,14 @@ has been destroyed.
   # PT supports request_store versions for 3 years.
   s.add_dependency "request_store", "~> 1.4"
 
+  # The following gems have been extracted from the Ruby stdlib to gems, and we
+  # must manually include them here in order for specs to run.
+  s.add_development_dependency "benchmark", "~> 0.4.0"
+  s.add_development_dependency "bigdecimal", "~> 3.1"
+  s.add_development_dependency "drb", "~> 2.2"
+  s.add_development_dependency "logger", "~> 1.6"
+  s.add_development_dependency "mutex_m", "~> 0.3.0"
```

Comment:
I'm not sure of the best way to determine which version specifiers to write here.

For each gem, I looked at the latest version available (at rubygems.org).

If the latest version has a major version of `0`, then I did `~> <latest version>`.

If the latest version has a non-zero major version, then I did `~> <major.minor of latest version>`.

I have little idea if this is an optimal algorithm, and would be happy to make any changes.